### ISSUE TITLE
Can I provide a callback when visible range changes? 

### ISSUE THREAD

**Issue by paulpilone** [wojtekmaj/react-calendar#18]
Thanks for the project! 

Is there a way to provide a callback when the visible range changes? In my case I'm have other components that rely on knowing the visible range (e.g. the month of December in the Month view) and update if the user navigates to the next or previous months.

Digging through the source it looks like the next and previous buttons set the `activeStartDate`, but there is no way to be notified that the `activeStartDate` has changed (e.g. `onActiveStartDateChange`). Am I missing something? Is this a feature you or other users might find useful?

Thanks!

**Comment by wojtekmaj** [comment]
Seems like an awesome idea. Entirely doable. Thanks!

**Comment by paulpilone** [comment]
@wojtekmaj Great! I'm actually using the library as more of a calendar display rather than a calendar picker (although the user can select a date). In my use case, it would make sense to set the selected date to the first or last day of the month depending on the navigation direction.

I'm not sure if that would be useful with respect to a date picker implementation. For me, just knowing that the visible range changes (which looks like the `activeStartDate`) changed is sufficient and would be completely backwards compatible.

**Comment by wojtekmaj** [comment]
Hmmm... I don't think I'd be able to provide you with such specific callbacks. 
But you will be provided with three new callbacks, `onDrillUp`, `onDrillDown`, `onActiveDateChange`, all returning an object with new `activeStartDate` and `view`. Should be generic enough to suit everyone's needs, and you could be tracking previous activeStartDate to figure out the direction.

```
var previousActiveStartDate = new Date(2017, 0, 1);
var activeStartDate = new Date(2017, 1, 1);

console.log(previousActiveStartDate < activeStartDate ? 'forward' : 'back');
```

**Comment by wojtekmaj** [comment]
Fixed in 2.8.0.

**Comment by paulpilone** [comment]
Thanks! Updated and it's working perfectly. Thanks for the quick turnaround!
